Source/WebKit/UIProcess/Cocoa/WebProcessProxyCocoa.mm

```diff
@@ -333,4 +333,12 @@ bool shouldTransformObject(id object) const override
     return connection()->getAuditToken();
 }
 
+SandboxExtension::Handle WebProcessProxy::fontdMachExtensionHandle(SandboxExtension::MachBootstrapOptions machBootstrapOptions)
+{
+    SandboxExtension::Handle fontMachExtensionHandle;
+    if (auto handle = SandboxExtension::createHandleForMachLookup("com.apple.fonts"_s, auditToken(), machBootstrapOptions))
```

Comment:
We may be able to write:
`return SandboxExtension::createHandleForMachLookup("com.apple.fonts"_s, auditToken(), machBootstrapOptions).value_or(SandboxExtension::Handle { });`

Reply:
> We may be able to write:
> `return SandboxExtension::createHandleForMachLookup("com.apple.fonts"_s, auditToken(), machBootstrapOptions).value_or(SandboxExtension::Handle { });`

Nice!